ios-base/DroidKaigi 2020/Session/Views/SessionViewController.swift

```diff
@@ -0,0 +1,99 @@
+import ios_combined
+import UIKit
+import RxSwift
+import RxCocoa
+import MaterialComponents
+
+final class SessionViewController: UIViewController {
+
+    private let disposeBag = DisposeBag()
+
+    @IBOutlet weak var filteredSessionCountLabel: UILabel! {
+        didSet {
+            filteredSessionCountLabel.font = ApplicationScheme.shared.typographyScheme.caption
+        }
+    }
+    @IBOutlet weak var filterButton: MDCButton! {
+        didSet {
+            filterButton.isSelected = true
+            filterButton.applyTextTheme(withScheme: ApplicationScheme.shared.buttonScheme)
+            let filterListImage = UIImage(named: "ic_filter_list")
+            let templateFilterListImage = filterListImage?.withRenderingMode(.alwaysTemplate)
+            filterButton.setImage(templateFilterListImage, for: .selected)
+            let arrowUpImage = UIImage(named: "ic_keyboard_arrow_up")
+            let templateArrowUpImage = arrowUpImage?.withRenderingMode(.alwaysTemplate)
+            filterButton.setImage(templateArrowUpImage, for: .normal)
+            filterButton.setTitle("Filter", for: .selected)
+            filterButton.setTitle("", for: .normal)
+        }
+    }
+    @IBOutlet weak var collectionView: UICollectionView! {
```

Comment:
Just out of my curiosity, why did you use `UICollectionView` instead of `UITableView`?

Reply:
At the time I implemented this, I would use [MDCBaseCell](https://material.io/develop/ios/components/list/api-docs/Classes/MDCBaseCell.html) or [MDCSelfSizingStereoCell](https://material.io/develop/ios/components/list/api-docs/Classes/MDCSelfSizingStereoCell.html). This is inherit class of `UICollectionViewCell`. So here uses `UICollectionView` .

Reply:
I completely understand. Thanks.